docs/Quickstart.md

```diff
@@ -43,28 +43,31 @@ git remote add heroku git@heroku.com:appinfive.git
 
 4. Add ShopifyApp to gemfile
 ----------------------------
-```
+```sh
 $ echo "gem 'shopify_app'" >> Gemfile
-
-bundle install
+$ bundle install
 ```
 
 Note - its recommended to use the latest released version. Check the git tags to see the latest release and then add it to your Gemfile e.g `gem 'shopify_app', '~> 7.0.0'`
 
 5. Run the ShopifyApp generator
 -------------------------------
+```sh
+# use the keys from your app in the partners area
+$ rails generate shopify_app --api_key a366cbafaccebd2f615aebdfc932fa1c --secret 8750306a895b3dbc7f4136c2ae2ea293
+$ git add .
+$ git commit -m 'generated shopify app'
 ```
-use the keys from your app in the partners area
-rails generate shopify_app --api_key a366cbafaccebd2f615aebdfc932fa1c --secret 8750306a895b3dbc7f4136c2ae2ea293
-git add .
-git commit -m 'generated shopify app'
-```
+
+If you forget to set your keys or redirect uri above you will find them in the shopify_app initializer at: `/config/initializers/.`
+
+We recommend adding a gem or utilizing ENV variables to handle your keys before releasing your app.
```

Comment:
I agree with this, and maybe it would be better to have the example above NOT show adding keys, which makes them added to `shopify_app.rb` initializer.

Perhaps change the above to:

`$ rails generate shopify_app --api_key <shopify_api_key> --secret <shopify_api_secret>`

I believe these are the default values in the generator itself if nothing is passed as command line argument.

Reply:
The defaults in the initializer are `<api_key>` and `<secret>` respectively. Do you think it'd be better to mirror those completely, or to use your values to be more descriptive? I'm leaning toward your recommendation, but interested to hear your thoughts. :D

Reply:
I'm indifferent. I just know that a lot of people just copy-paste everything so it might be good to use obviously dummy values and point them in the right direction (which might even be pointing them towards gems like [figaro](https://github.com/laserlemon/figaro) or [dotenv](https://github.com/bkeepers/dotenv)

Reply:
I think this is a good suggestion 👍

Reply:
Done and done.